### DISCUSSION TITLE
指令跟随效果很差

### DISCUSSION THREAD

**Discussion by renyangang** [zai-org/ChatGLM3#450] [Bad Case]
例如：
在system提示中如下规定
你的输出的论文章节内容需要遵守以下规定：
1. 针对每章节的输出，以写作者身份直接输出生成的内容，不要添加提示语句，不要重复用户输入的章节标题或者描述。
在user提示：
请完成论文中 "2.3 B/S架构" 这一章节的内容编写并输出, 输出一段文字描述，不要包含子标题,首行不要出现输入的标题内容，不要编造人名和文献时间等内容, 不要在输出内容中增加类似"本章节.." "在本章中.." 这样的提示。

在输出中，仍然带有章节标题：
2.3 B/S架构
  B/S（Browser/Server）架构，即浏览器/服务器模型，是一种典型的 Web 应用开发模式。在该架构中，客户端通过浏览器访问服务器，服务器负责提供服务和数据处理。这种架构具有跨平台、易于维护和扩展的特点，已经成为Web应用开发的主流架构之一。

**Comment by zRzRzRzRzRzRzR** [comment]
> 例如： 在system提示中如下规定 你的输出的论文章节内容需要遵守以下规定：
> 
>     1. 针对每章节的输出，以写作者身份直接输出生成的内容，不要添加提示语句，不要重复用户输入的章节标题或者描述。
>        在user提示：
>        请完成论文中 "2.3 B/S架构" 这一章节的内容编写并输出, 输出一段文字描述，不要包含子标题,首行不要出现输入的标题内容，不要编造人名和文献时间等内容, 不要在输出内容中增加类似"本章节.." "在本章中.." 这样的提示。
> 
> 
> 在输出中，仍然带有章节标题： 2.3 B/S架构 B/S（Browser/Server）架构，即浏览器/服务器模型，是一种典型的 Web 应用开发模式。在该架构中，客户端通过浏览器访问服务器，服务器负责提供服务和数据处理。这种架构具有跨平台、易于维护和扩展的特点，已经成为Web应用开发的主流架构之一。

尝试在系统指令中输入，用户的指令跟随实测不如系统指令

**Reply by renyangang** [reply]
系统指令和用户指令都尝试过，效果都不稳定

**Reply by zRzRzRzRzRzRzR** [reply]
嗯，这个情况是存在的， 属于badcase，下一代模型会尽量解决这个问题

**Comment by chopin1998** [comment]
> 嗯，这个情况是存在的， 属于badcase，下一代模型会尽量解决这个问题

下一代模型什么时候出？ 能不能搞一个 chatGLM4 13b的
